code/modules/surgery/surgery.dm

```diff
@@ -131,7 +131,7 @@ proc/do_surgery(mob/living/carbon/M, mob/living/user, obj/item/tool)
 		if(success == TOOL_USE_SUCCESS)
 			selectedStep.end_step(user, M, zone, tool)		//finish successfully
 		else if(success == TOOL_USE_FAIL)
-			tool.handle_failure(user, M, required_stat = STAT_BIO, selectedStep.requedQuality)
+			tool.handle_failure(user, M, required_stat = STAT_BIO, required_quality = selectedStep.requedQuality)
```

Comment:
This was throwing a runtime error that positional qualities must come before named qualities